src/tox_lsr/test_scripts/runwoke.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Do not exit on an error to continue ansible-doc and ansible-test.
+set -euo pipefail
+
+#uncomment if you use $ME - otherwise set in utils.sh
+#ME=$(basename "$0")
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+. "${SCRIPTDIR}/utils.sh"
+
+curl -sSfL https://git.io/getwoke | bash -s -- -b "$LSR_TOX_ENV_DIR/bin"
+WOKE_CMD="$LSR_TOX_ENV_DIR/bin/woke"
+WOKE_OUT="$LSR_TOX_ENV_DIR/tmp/woke_output.txt"
+
+trap "rm -f $TOXINIDIR/.woke.yml" RETURN
+
+pushd "$TOXINIDIR" > /dev/null
```

Comment:
You shouldn't have to pushd or cd to `$TOXINIDIR` because the testenv:woke does `changedir = {toxinidir}`